allensdk/brain_observatory/ecephys/ecephys_project_api/ecephys_project_lims_api.py

```diff
@@ -203,6 +204,13 @@ def and_filters(filters):
 def clean_wkf_response(response):
     if response.shape[0] == 0:
         return response
-    response['path'] = response.apply(lambda row: os.path.join(row['storage_directory'], row['filename']), axis=1)
+
+    def build_path(row):
+        return os.path.join(
```

Comment:
Is internal going to be supporting 2.7 going forward? If not, might be worth considering using `pathlib` unless `build_path()` is really performance critical.

Reply:
I switched all of ecephys to pathlib. Not convinced it's better but would appreciate your thoughts. See here for discussion:
https://github.com/AllenInstitute/AllenSDK/issues/675